x-pack/legacy/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/CustomLinkFlyout/FiltersSection.tsx

```diff
@@ -88,7 +89,7 @@ export const FiltersSection = ({
           <EuiFlexGroup key={filterId} gutterSize="s" alignItems="center">
             <EuiFlexItem>
               <EuiSelect
-                aria-label={filterId}
+                data-testid={filterId}
```

Comment:
FWIW, maybe you know it, then ignore this comment, but in EUI there is convention to use `data-test-subj` for this.

Reply:
Thanks @streamich I didn't know that! 👍🏻